spinalcordtoolbox/csa_pmj.py

```diff
@@ -33,7 +32,7 @@ def get_slices_for_pmj_distance(segmentation, pmj, distance, extent, param_cente
     im_seg.change_orientation('RPI')
     im_pmj = Image(pmj).change_orientation('RPI')
     if not im_seg.data.shape == im_pmj.data.shape:
-        raise RuntimeError(f"segmentation and pmj should be in the same space coordinate.")
+        raise RuntimeError(f'{"segmentation and pmj should be in the same space coordinate."}')
```

Comment:
Same sort of comment -- can we fix these linting issues in @jcohenadad's branch, that way we won't need to do it here?

Reply:
Ah, I should also mention: Instead of adding `{` to the string, I think you can just get rid of the `f""` entirely and make it a normal string.

Reply:
yes on it!